src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -80,6 +80,25 @@ def sql_unset_upgraded_to(self, catalog):
             f"UNSET TBLPROPERTIES IF EXISTS('upgraded_to');"
         )
 
+    def is_dbfs_root(self) -> bool:
+        if self.location is None:
+            return False
+        if self.location.startswith("/dbfs/mnt"):
```

Comment:
Nit: Can you create a "dbfs_root_prefixes" list and iterate over it?

Reply:
Sure, makes sense. I was thinking just that.